events/fair-ml.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: "Fair ML Speaker Series"
+permalink: "/fair-ml"
+---
+
+<h1 class="title page-title"> Fair ML Speaker Series</h1>
+
+<p style = "font-size: 115%;">
+	Thanks to our <a class="tla-link" href="{{site.baseurl}}/speaker-series"> Speaker Series Team </a>, our Fair ML Speaker Series was a success! <b> Learn all about it on this page. </b>
+
+</p>
+
+<hr class="divider" />
+
+<div class="citylab-section" style = "margin-top: 2em;">
```

Comment:
I would maybe swap the order of this section and the predictive policing one - from a framing perspective, this is a "more resources"/credits type of content block, while the other is the main content. We can also discuss how to revise the copy later!